runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/control/SdkHarnessClient.java

```diff
@@ -412,6 +387,34 @@ public void abort() {
     return bundleProcessor;
   }
 
+  /**
+   * A {@link StateDelegator} that issues zero state requests to any provided
+   * {@link StateRequestHandler state handlers}.
+   */
+  private static class NoOpStateDelegator implements StateDelegator {
```

Comment:
Why is this moved?

Reply:
The move is actually for "public BundleProcessor getProcessor.........." but the diff creator picked it as this move.
G3 Linter wants all the overloaded methods to be together.